src/stores/TransformationCreate/Store.ts

```diff
@@ -1,33 +1,67 @@
 import produce from 'immer';
-import { intersection } from 'lodash';
+import { intersection, omit } from 'lodash';
 import { TransformCreateStoreNames } from 'stores/names';
+import { hasLength } from 'utils/misc-utils';
 import { devtoolsOptions } from 'utils/store-utils';
 import { create, StoreApi } from 'zustand';
 import { devtools, NamedSet } from 'zustand/middleware';
 import { TransformCreateState } from './types';
 
+const evaluateSelectedAttribute = (
```

Comment:
Please add some comments to this function to walk through what is happening in it.

Reply:
I will not at present because the function will be removed shortly.